src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -41,6 +41,8 @@ public class ParameterProvider {
   public static final String BDEC_PARQUET_COMPRESSION_ALGORITHM =
       "BDEC_PARQUET_COMPRESSION_ALGORITHM".toLowerCase();
 
+  public static final String ENABLE_CHUNK_ENCRYPTION = "ENABLE_CHUNK_ENCRYPTION".toLowerCase();
```

Comment:
I don't think we want this to be a configurable parameter

Reply:
+1 on not allowing customers to configure this.

Reply:
also pointed this out in the previous PR: https://github.com/snowflakedb/snowflake-ingest-java/pull/768#discussion_r1637199029 !